src/components/Footer/Footer.jsx

```diff
@@ -1,4 +1,6 @@
 import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
```

Comment:
@Amruta7203 You are using these dependencies. But there is no update with package.json. And I think that's where we are confusing the build in believing that there is a dependency or not for react-fontawesome. 

It should be working in your system because you may have it.

You gotta do

```bash
git add src/components/Footer/Footer.jsx package.json
```

Also to add more, You have replaced the text with Icons but they look empty. Have a text besides them too.